cypress/fixtures/extensions/extension-foo/scripts/foo.js

```diff
@@ -0,0 +1,20 @@
+
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
```

Comment:
Do we want extension authors to use the same namespace for JS as we do? 🤔

Reply:
I'll change window.GOVUK to window.FOO